### PR TITLE
Edited the groomer dashboard feature to allow groomers to upload and …

### PR DESCRIPTION
**Pull request by broph123**
## Description ##

The feature I updated relates to Groomer in their dashboard being able to upload their grooming license allowing them to operate. This aspect of the groomer dashboard is important because groomers need to be able to manage their licensing paperwork in order to operate their business. Customers also feel safer knowing that their groomer is licensed and can see the uploaded paperwork in their customer dashboard. 

Before I made changes, the groomer could upload an image and only the file name would render. After I made changes, I added to the functionality allowing not only the file name to render but also a smaller version of the document. I also added the ability for groomers to click on the picture to enlarge it. 

There is still some work to be done. The backend and this feature update are not connected. Also the image uploader only accepts .jpg and .pngs right now. I tried uploading a PDF and it wouldn't render on screen. 

user story link: https://trello.com/c/J63WiV48/2-as-a-groomer-i-can-add-my-groomer-license-number-and-upload-a-copy-of-my-license

## Type ##

- [ ] This is a bugfix.
- [ ] This is a new feature.
- [ X] This is part of a feature.

## Organization ##

- [X ] Have you linked this pull request to a Trello card?
- [ ] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 
## Review ##

- [X ] Is this a work in progress?
- [ ] Is this ready to be reviewed?
- [ X] Have you added two reviewers to this pull request?

### CODE REVIEW

File: src/components/common/FileUpload.js [hunk 1]
```diff
@@ -1,40 +1,41 @@
 import React, { useState } from 'react';
-import axios from 'axios';
-import { Button } from 'antd';
+// import axios from 'axios';
```
Comment (cearc-sara) [inline src/components/common/FileUpload.js]: why is this commented out? Does it need to be there? If so please leave a comment next to it as to why its commented out.

File: src/components/common/FileUpload.js [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+  // const { authState } = useOktaAuth();
   const [selectedFile, setSelectedFile] = useState();
 
-  const getAuthHeader = authState => {
-    if (!authState.isAuthenticated) {
-      throw new Error('Not authenticated');
-    }
-    return { Authorization: `Bearer ${authState.idToken}` };
-  };
+  // const getAuthHeader = authState => {
```
Comment (cearc-sara) [inline src/components/common/FileUpload.js]: same with all the code in this file, if it needs to be commented out, leave a comment next to them as to why, if they do not need to be there then just delete them.

**Review by cearc-sara — CHANGES_REQUESTED** [review]
Please review the comments I left throughout the code, and make the appropriate changes.

File: .env.sample [hunk 3]
```diff
@@ -1,5 +0,0 @@
-REACT_APP_CLIENT_ID=example
```
Comment (cearc-sara) [inline .env.sample]: yeah this is the example that needs to be in the github, there isn't anything in there that is specific to our project. So just add that back and then push again and you should be good to go.

**Review by cearc-sara — CHANGES_REQUESTED** [review]
Just review the comment I left and let me know when you have changed it.

**Review by cearc-sara — APPROVED** [review]

**Review by vishalicious213 — APPROVED** [review]
The groomer license components look fine to me. You're good to go!

**Pull request merged**